gen3-integration-tests/pyproject.toml

```diff
@@ -35,6 +37,8 @@ markers = [
   "mds: tests for metadata-service",
   "portal: tests for data-portal",
   "workspace: tests for workspace page",
+  "sheepdog: tests for sheepdog",
```

Comment:
I would like to avoid using the specific service names as much as possible in these tests. Ideally I'd like these markers to focus on specific features (that may happen to be all from 1 service) rather than the other way around.

That way the tests are testing at the feature abstraction so if we change what services we use for a given feature in the future, we don't have so many places to update here. So maybe something like: `StructuredData` for Sheepdog or something and maybe `StructuredDataGraphQL` for peregrine?

Reply:
I agree, but i figured this might be a way for the QA team to set up some automation around which tests are run on which repo, like in the old pipeline with `@require-xyz` labels. If not, feature-level is definitely better. @atharvar28 @haraprasadj do you have any insight?

Reply:
It still can do that without using the service names. We talked briefly about this before. We just need to make sure that the markers a service supports is listed in the repo. e.g. the abstraction is:

- a repo is likely a service which support n features
- each feature has a marker
- each feature has tests

By labeling repo with n markers, we can ensure the tests affected are run

Reply:
it's fine if n == 1, I just want to make sure the abstraction is there so the service name itself isn't on the test side of things

Reply:
fair enough, i'll update it

Reply:
yes, let's have additional markers for feature itself but retain the service-level features as that is a way to set up filtering of tests to run on each repo